lib/core/base/audit.js

```diff
@@ -40,8 +40,191 @@ function Audit(audit) {
 
 	this.defaultConfig = audit;
 	this._init();
+
+	// A copy of the "default" locale. This will be set if the user
+	// provides a new locale to `axe.configure()` and used to undo
+	// changes in `axe.reset()`.
+	this._previousLocale = null;
 }
 
+/**
+ * Build and set the previous locale. Will noop if a previous
+ * locale was already set, as we want the ability to "reset"
+ * to the default ("first") configuration.
+ */
+
+Audit.prototype._setPreviousLocale = function() {
```

Comment:
Not a huge deal, but I'd prefer we use named functions where we can in axe-core. It tends to make debugging a lot easier.

Reply:
I think you're mistaking this method for a function expression. Because this is a method, we'll see its implicit name in stack traces.

Given the code:

```js
function F () {}

F.prototype.foo = function () {
  throw new Error('boom')
}

const f = new F
f.foo()
```

We'll clearly see "foo" in the stack trace:

```
node foo.js
/Users/stephen/dev/src/github.com/dequelabs/axe-core/foo.js:4
  throw new Error('boom')
  ^

Error: boom
    at F.foo (/Users/stephen/dev/src/github.com/dequelabs/axe-core/foo.js:4:9)
    at Object.<anonymous> (/Users/stephen/dev/src/github.com/dequelabs/axe-core/foo.js:8:3)
    at Module._compile (internal/modules/cjs/loader.js:702:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:713:10)
    at Module.load (internal/modules/cjs/loader.js:612:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:551:12)
    at Function.Module._load (internal/modules/cjs/loader.js:543:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:744:10)
    at startup (internal/bootstrap/node.js:238:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:572:3)
```